utils_cv/multi_object_tracking/display_with_bb.py

```diff
@@ -0,0 +1,123 @@
+import cv2
+import pandas as pd
+
+def convert_trackingbboxes_txt(prediction_dict, result_file_path) -> None: # called by evaluate() in TrackingLearner class
```

Comment:
[minor] the filename could be more selfexplanatory. Maybe rename to something like: convert_to_fairmot_annotation()